lua/damagelogs/shared/autoslay.lua

```diff
@@ -119,6 +132,16 @@ local function CreateCommand()
 end
 hook.Add("Initialize", "AutoSlay", CreateCommand)
 
+hook.Add( "TTTEndRound", "AutoSlay_CheckForMarked", function( res )
```

Comment:
Hook should be called only on the server side.

Reply:
Thought I wrapped it in `if SERVER then` but I guess I forgot.